CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ It diverges in the following ways:
 ### Changed
 
 ### Fixed
+- Partial fix for poorly formatted videos (#2386)
```

Comment:
Can you be more specific? Why are they poorly formatted and what did we fix?

Reply:
I've normally kept these short, assuming that if someone wanted more information they would just look at the commit message.  But I can expand it.

Reply:
I think they should be short but informative: ideally, a person who is experiencing this bug can read this and know if it addressed their specific fullscreen problem or another one.

Reply:
If I recall correctly, this only resolves the issue for 2 out of 13 sites where it's experienced, so that's unfortunately going to be next to impossible in this case.  I did add some more detail though.